File: packages/core/test/fixtures/malicious/python/exec-base64/input.py
"""Defanged: exec() of a base64-decoded blob."""
import base64
# Real-world attacks would put the actual payload here.
# This blob decodes to: print("stub")
_BLOB = "cHJpbnQoInN0dWIiKQ=="
exec(base64.b64decode(_BLOB).decode())

File: packages/core/test/fixtures/malicious/perl/eval-base64/input.pl
# Defanged: classic Perl decode-then-eval webshell shape.
# Decodes to: print "stub";
use MIME::Base64 qw(decode_base64);
my $blob = "cHJpbnQgInN0dWIiOw==";
eval(decode_base64($blob));
